docs/sphinx/source/installation.rst

```diff
@@ -3,6 +3,12 @@
 Installation
 ============
 
+Compatibility
+~~~~~~~~~~~~~
+
+bifacial_radiance is coded and tested in Windows, but can also work on Linux and Mac OSX, particularly after improvements in the latest release :ref:`whatsnew_0302` which solved some of the binary issues for gencumsky. However, the functionalities in Linux are still being improved, for example the GUI requires special QT installation described in (:issue:`130`:).
+
+
 Video Instructions
```

Comment:
Please, indicate somewhere that these video instructions are for Windows only.

Reply:
`https://youtu.be/4A9GocfHKyM <https://youtu.be/4A9GocfHKyM>`_ This video shows how to install the bifacial_radiance software and all associated softwares needed for Windows. More info on the Wiki. Instructions for Windows and Linux-based OS are also shown below.

Also updated on info of the youtube video and title of the youtube video